ACE/ace/config-linux.h

```diff
@@ -403,6 +403,7 @@
 #define ACE_HAS_RECURSIVE_THR_EXIT_SEMANTICS
 #define ACE_HAS_2_PARAM_ASCTIME_R_AND_CTIME_R
 #define ACE_HAS_REENTRANT_FUNCTIONS
+#define ACE_HAS_MNTENT
```

Comment:
I suppose it could, but since it is dictating whether the dependent code can use struct mntent and related functions, it is valid the way it is.